### ISSUE TITLE
Enhancement - Add import of packages to install from packages.config

### ISSUE THREAD

**Issue by gep13** [chocolatey/ChocolateyGUI#248]
Related to #7 

Should be able to click an import button, select a packages.config file, and have all defined packages in that file installed, including specific version.

For now, let's not worry about additional parameters, i.e. installation arguments, source, etc. let's just get something out there.

**Comment by LinkofHyrule89** [comment]
This should definitely be a feature it would help a ton with either fresh installs of Windows, new computers, or just when I refresh my computer.

**Comment by gep13** [comment]
@LinkofHyrule89 I agree.  It is currently supported through the choco command line, so it shouldn't take much to expand to include the functionality in chocolateygui, I just need to get some time to come back around to this.  Definitely still on the radar.

**Comment by EntranceJew** [comment]
I'm not sure why this is still marked as "not planned", with the "export" button so prominent on the UI as of `0.13.2.0` -- seems weird that everyone savvy enough to import would even need an export button. If there's no plan for implementing this then I don't understand the point of having the export still.

**Comment by mwallner** [comment]
Thanks for bringing this up again @EntranceJew - don't know if it makes it into the next release that I'm trying to "push" - but it's definitely a feature worth having!

**Comment by RichiCoder1** [comment]
I actually though I stripped out the export button in 0.14+. Apparently I forgot too.
